content/foundations/color.mdx

```diff
@@ -2,64 +2,187 @@
+| `fg.muted`      | Use for content that is secondary or provides additional context but is not critical to understanding the flow of an interface.  | 
+| `fg.onEmphasis` | On emphasis is the text color designed to combine with -emphasis backgrounds for optimal contrast.                               | 
+
+| Color roles    | Purpose                                                            | 
+| :--------------| :------------------------------------------------------------------| 
+| `fg.accent`    | Use for interactive text or icons.                                 | 
+| `fg.success`   | Use to emphasize a positive message.                               | 
+| `fg.attention` | Use to flag text or icons that require the users attention         | 
+| `fg.danger`    | Use to emphasize an error or a blocking status. Action is required | 
+| `fg.severe`    | Use to emphasize a level of severity between attention and danger  | 
+| `fg.done`      | Use to style text that refers to completed tasks or workflows      | 
+
+### Backgrounds
+Background colors apply to surfaces of components or UI elements, such as pages, boxes, and overlays.
+
+| Foundations     | Purpose                                                                                                                  | 
+| :---------------| :----------------------------------------------------------------------------------------------------------------------- | 
+| `bg.default`    | Primary background color.                                                                                                | 
+| `bg.subtle`     | Provide visual rest and contrast against the default background.                                                         | 
+| `bg.onEmphasis` | It’s highest in contrast against the default background, such as in tooltips.                                            | 
+| `bg.inset`      | Can be used instead of default background when you want to create a focal point like in conversations or activity feeds. | 
+
+| Color roles             | Purpose                                                                       | 
+| :--------------------| :---------------------------------------------------------------------------- | 
+| `bg.accent`          | Use to accentuate interactive areas in the UI like selected elements.         | 
+| `bg.success`         | Use to highlight a positive message.                                          | 
+| `bg.attention`       | Use to highlight elements that require a users attention or pending statuses. | 
+| `bg.danger`          | Use to emphasize an error or a blocking status. Action is required.           | 
+| `bg.severe`          | Use to emphasize an extra level of severity between attention and danger.     | 
+| `bg.done`            | Use to style text that refers to completed tasks or workflows.                |
+| `bg.[ROLE].emphasis` | Use to highlight the most important item of a view or an interface            | 
```

Comment:
Not sure what [ROLE] is here.

Reply:
It's any of the roles mentioned above: accent, success, etc. I can't find a better way to be able to have in one line instead of making the full list of bg.success.emphasis, bg.accent.emphasis... any suggestions?

Reply:
Wondering if it could be [ANY_OF_ABOVE].emphasis. Not sure this would be better though. 🙈